easybuild/tools/modules.py

```diff
@@ -284,7 +285,7 @@ def set_and_check_version(self):
 
     def check_cmd_avail(self):
         """Check whether modules tool command is available."""
-        cmd_path = which(self.cmd)
+        cmd_path = which(self.cmd, log_ok=False)
```

Comment:
What's the point of disabling the logging here?

Reply:
Because logging is done 3 lines below

Reply:
OK